src/app/states/editing-note/page.mdx

```diff
@@ -0,0 +1,44 @@
+export const metadata = {
+  title: 'editingNote',
+  description: 'An app state of editingNote in Inkdrop. This section explains how to fetch the `editingNote` state.',
```

Comment:
I think it is better to explain a word without having to use the very word that is to be explained.
```suggestion
  description: 'An app state of currently opened note in the editor. This section explains how to fetch the `editingNote` state.',
```